docs/ref/settings.txt

```diff
@@ -9,6 +9,14 @@ default values. All settings are prefixed with ``FILTERS_``, although this
 is a bit verbose it helps to make it easy to identify these settings.
 
 
```

Comment:
@rpkilby Can I just ask you to add

* A sub/section header grouping these `help_text` related settings
* A paragraph highlighting the out-of-the-box "Off but with BaseCSV default" behaviour, which leads to needing `FILTERS_DISABLE_HELP_TEXT`

Thanks.